test/parallel/test-worker-eval-typescript.js

```diff
@@ -19,25 +19,31 @@ const cjsHelloWorld = `
 
 const disableTypeScriptWarningFlag = '--disable-warning=ExperimentalWarning';
 
-test('Worker eval module typescript without input-type', async () => {
-  const w = new Worker(esmHelloWorld, { eval: true, execArgv: [disableTypeScriptWarningFlag] });
-  assert.deepStrictEqual(await once(w, 'message'), ['Hello, World!']);
-});
+test('Worker eval module typescript without input-type',
+     { skip: !process.config.variables.node_use_amaro },
+     async () => {
```

Comment:
You could make the diff more readable by using a variable
```suggestion
const onlyWithAmaro = { skip: !process.config.variables.node_use_amaro };

test('Worker eval module typescript without input-type', onlyWithAmaro, async () => {
```